src/terminal7.js

```diff
@@ -183,8 +183,8 @@ export class Terminal7 {
                 })
         // hide the modal on xmark click
         // Handle network events for the indicator
-        Network.addListener('networkStatusChange', s => 
-            this.updateNetworkStatus(s))
+        /* Network.addListener('networkStatusChange', s => 
+            this.updateNetworkStatus(s)) */
```

Comment:
why?

Reply:
This line causes the app to constantly refresh, I don't know why the event is being called.

Reply:
It''s used to identify when the network is down

Reply:
![image](https://user-images.githubusercontent.com/37512471/218801572-f9390faa-62e3-43a0-a764-e78e9c724917.png)
This is what happens when this check is uncommented (the host is online).

Reply:
We need to improves what happens on network down, but that's a separate issue. 
For now try to protect it against reentrancy. Try adding a `lastNetworkState` and call the handler nly when it's a state change.